docs/quick-visual-tutorials/visual-ibac-istio-authorization-policies.mdx

```diff
@@ -8,15 +8,322 @@ import CodeBlock from "@theme/CodeBlock";
+With IBAC, you won't need to manually keep track of pod labels or service accounts, and we'll even show you how to generate policies for all traffic in the cluster with just one command.
+In the end of this tutorial each service in the cluster will be able to communicate only with the servers declared in its client intents file, blocking any unintentional communication.
 
-This visual tutorial for IBAC with Istio authorization policies is not yet ready.
+All the capabilities of IBAC are within Otterize OSS, while the access graph in Otterize Cloud will guide us visually in these steps.
+We will:
+1. Install Otterize on your cluster
+2. View the traffic in your cluster in the cloud
+3. Declare one intent just to see how its generated policies effect the cluster
+4. Generate intents for all traffic in the cluster to achieve zero-trust in one command
 
-In the meantime, you might want to check out some similar tutorials:
-* [IBAC with network policies](/quick-visual-tutorials/visual-ibac-network-policies)
-* [IBAC with Kafka in Kubernetes](/quick-visual-tutorials/visual-ibac-kafka-k8s).
+## Prerequisites
 
-And for a simple, less visual tutorial, see:
-* [Istio AuthorizationPolicy automation with intents](/quick-tutorials/k8s-istio-authorization-policies)
+<details>
+<summary>Prepare a Kubernetes cluster</summary>
 
+Before you start, you'll need a Kubernetes cluster. Having a cluster with a [CNI](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/) that supports [NetworkPolicies](https://kubernetes.io/docs/concepts/services-networking/network-policies/) isn't required for this tutorial, but is recommended so that your cluster works with other tutorials.
+
+{@include: ../_common/cluster-setup.md}
+</details>
+
+<details>
+<summary>Create an Otterize Cloud account</summary>
+
+{@include: ../_common/create-account.md}
+
+</details>
+
+<details>
+<summary>Install Otterize OSS <b>with Istio watcher active</b></summary>
+
+{@include: ../_common/install-otterize-from-cloud-with-istiowatcher.md}
+
+</details>
+
+<details>
+<summary>Install the Otterize CLI</summary>
+
+{@include: ../_common/install-otterize-cli.md}
+
+</details>
+
+## Install and configure Istio
+<details>
+<summary>Install Istio in the cluster via Helm</summary>
+
+{@include: ../_common/install-istio.md}
+
+</details>
+
+<details>
+<summary>Add HTTP methods and request paths to Istio exported metrics</summary>
+
+Apply this configuration in the `istio-system` namespace, propagating it to all namespaces covered by the mesh.
+
+```
+kubectl apply -f https://docs.otterize.com/code-examples/network-mapper/istio-telemetry-enablement.yaml
+```
+
+```yaml
+{@include: ../../static/code-examples/network-mapper/istio-telemetry-enablement.yaml}
+```
+
+</details>
+
+:::tip
+HTTP request paths and methods aren't exported in Envoy's connection metrics by default, but we do want to capture those
```

Comment:
Start with explain that we want to view HTTP resource and it's isn't enabled by default